DXRMissions/DeusEx/Classes/DXRMissionsM08.uc

```diff
@@ -157,8 +176,12 @@ function AnyEntry()
 
 function AfterMoveGoalToLocation(Goal g, GoalLocation Loc)
 {
+    local bool RevisionMaps;
+
+    RevisionMaps = class'DXRMapVariants'.static.IsRevisionMaps(player());
+
     if (g.name=="Jock") {
-        if(Loc.name=="Smuggler Back Entrance" || Loc.name=="Alley") {
+        if((!RevisionMaps && Loc.name=="Smuggler Back Entrance") || Loc.name=="Alley") {
```

Comment:
so Revision gets small Jock for the alley but not for the smuggler back entrance? I guess that area has more space in Revision?

Reply:
Yeah, there's a big open space there for a full-sized Jock.  I did have Small Jock originally, but it looked weird